components/Content.js

```diff
@@ -3,19 +3,53 @@ import Link from 'components/Link'
 import parse, { domToReact } from 'html-react-parser'
 
 export default ({ content }) => (
-  <article className="prose max-w-none mb-8 last:mb-0 break-words">
+  <article className="max-w-none last:mb-0 break-words leading-7">
```

Comment:
I wouldn't put `leading-7` on all text elements, just on the paragraphs because for most elements `leading-tight` works best. And I prefer `leading-relaxed`, but maybe that is just taste.